estimagic/dashboard/plot_functions.py

```diff
@@ -37,6 +37,10 @@ def plot_time_series(
     plot = create_styled_figure(
         title=title, name=name, logscale=logscale, plot_width=plot_width
     )
+    # this ensures that the y range spans at least 0.1
+    plot.y_range.range_padding = 0.05
```

Comment:
Please move to `estimagic.config`

Reply:
I moved all style settings to the config in fbbe96d.